Moblin/Various/Model.swift

```diff
@@ -4678,9 +4694,8 @@ extension Model: RemoteControlStreamerDelegate {
             }
             var topRight = RemoteControlStatusTopRight()
             if self.isShowingStatusAudioLevel() {
-                let level = formatAudioLevel(level: self.audioLevel) +
-                    formatAudioLevelChannels(channels: self.numberOfAudioChannels)
-                topRight.audioLevel = RemoteControlStatusItem(message: level)
+                topRight.audioLevel = self.audioLevel.isNaN ? nil : self.audioLevel
```

Comment:
What is the reason for assigning `self.audioLevel` conditionally?

Reply:
The reason is, when audio is muted (audiolevel = .nan) updateRemoteControlAssistantStatus won't get called. The reason seems to be that nan is noch decodeable/encodeable. took me a while to figure it out.

Reply:
What about using an enum to represent the audio level? I have not tested it, but should work. And makes it more clear than using `nil`. Does infinity (as in unknown) work when encoding a float using JSON? Probably not?
``` swift
enum AudioLevel: Codable {
    case muted
    case unknown
    case value(Float)
}
```

Reply:
it works :)